app/controllers/api/v0/instances_controller.rb

```diff
@@ -9,6 +9,7 @@ def show
           description: Settings::Community.community_description,
           logo_image_url: Settings::General.logo_png,
           name: Settings::Community.community_name,
+          registered_users_count: User.count,
```

Comment:
This is fine for now as most of the Forem instances will probably stay relatively small for the foreseeable future. But we do have `estimated_count` defined in `ApplicationRecord` which is much faster for larger tables than Postgres' default counting which always uses a full-table scan.

Reply:
Given the name is `registered_users_count`, this should be `User.registered` at least

I would consider switching to `.estimated_count` as well, which is the count we display publicly, unless a very precise count is needed for specific reasons, see also:

https://github.com/forem/forem/blob/49fb50e97915f5a8b81825f1f9344dcc6b2b694a/app/helpers/application_helper.rb#L268-L270

which is the count we display in the homepage when one is not authenticated